src/main/java/store/mybookstore/dto/book/CreateBookRequestDto.java

```diff
@@ -0,0 +1,26 @@
+package store.mybookstore.dto.book;
+
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import java.util.Set;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+    @NotNull(message = "Field title can't be empty!")

```

Comment:
It is better to use the `@NotBlank`
```suggestion
    @NotBlank(message = "Field title can't be empty!")
```